plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions.php

```diff
@@ -0,0 +1,210 @@
+<?php
+declare(strict_types=1);
+
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+use Automattic\WooCommerce\Admin\Features\Features;
+use Automattic\WooCommerce\Blocks\Utils\StyleAttributesUtils;
+
+/**
+ * CatalogSorting class.
```

Comment:
This needs to be updated. :slightly_smiling_face:

Reply:
Good eyes! Fixed on [f60e11d](https://github.com/woocommerce/woocommerce/pull/53215/commits/f60e11d468e449ee1a2f6f76cbd009a478a827f0)